src/components/Home.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react';
+import { CiMicrophoneOn, CiSettings } from 'react-icons/ci';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { categories } from './data';
+import {
+  getForex, getCryptos, getStocks, getEFTs,
+} from '../redux/actions/markets';
+import '../styles/Home.css';
+
+const Home = () => {
+  const forex = useSelector((state) => state.forex);
+  const cryptos = useSelector((state) => state.cryptos);
+  const stocks = useSelector((state) => state.stocks);
+  const efts = useSelector((state) => state.efts);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!forex.length) {
+      dispatch(getForex());
+    }
+
+    if (!cryptos.length) {
+      setTimeout(() => dispatch(getCryptos()), 2000);
+    }
+
+    if (!stocks.length) {
+      dispatch(getStocks());
+    }
+
+    if (!efts.length) {
+      dispatch(getEFTs());
+    }
+  }, [cryptos.length, dispatch, efts.length, forex.length, stocks.length]);
+
+  return (
+    <section className="home-page">
+      <div className="top-bar">
```

Comment:
- [ ] good job @Feliverse  in this page a user should be able to filter the crypto list either by name or price, kindly add a filter or search bar kind of component where a user can search for a particular crypto :+1:

Reply:
@VICTOR-CHUKWU thank you for the suggestion, I will consider it implemented the next week, for now, it is not a requirement for the capstone.